### ISSUE TITLE
s6 natpmp service broken

### ISSUE THREAD

**Issue by 4s1** [fusetim/external_natpmp_qbittorrent#5]
linuxserver-mod moved to [s6 v3](https://github.com/linuxserver/docker-mods/blob/master/README.md#new-v3-mods), so older (deprecated) s6 v2 mods are ignored (read: [get deleted]( https://github.com/linuxserver/docker-mods/blob/d63c461360c184002e0d02a230c1c9dbc7a501e4/docker-mods.v3#L419)).

So `etc/services.d/natpmp` needs to be migrated to something like `etc/s6-overlay/s6-rc.d/svc-natpmp/run`

**Comment by fusetim** [comment]
You are right, this mod is currently obsolete. I have currently no plan for upgrading it to be a S6v3 compatible mod. I am unsure if I will get the time to do it promptly or not.

Nonetheless PR are accepted and it might be more easy for me to find time to test it than to update it from scratch.
